source/public/javascripts/game.js

```diff
@@ -0,0 +1,21 @@
+$(document).ready(function(){
+
+  $('#player1-container').on('change', function(event){
+    $('#player1-container').toggle(false);
+    $('#player2-container').toggle(true);
+  });
+
+  $('#player2-container').on('change', function(event){
+    // $('#MyGame').submit();
+
+    // $target = $(event.target)
```

Comment:
That inherently is why your done is never getting called, because those values don't exists.